async-development/API.js

```diff
@@ -0,0 +1,14 @@
+let btn = document.getElementById('btn');
+let form = document.getElementById('form');
+form.addEventListener('submit', function() {
+    let city = document.getElementById('input').value;
```

Comment:
`let city = document.getElementById('input').value;`
You could declare it in the upper scope rather than find it on every submit event.
Do not forget, that interacting with DOM is an expensive operation